lib/pengine/pe_notif.c

```diff
@@ -633,13 +625,22 @@ collect_resource_data(pe_resource_t *rsc, bool activity, notify_data_t *n_data)
     }
 }
 
+// For (char *) value
```

Comment:
We still use `add_notify_env()` in `pe__order_notifs_after_fencing()`, and there's no reason to use a `GString` there. It felt most straightforward to use separate macros for regular strings and `GString`s.

No need for a regular string macro for `add_notify_env_free_gs()` but it seemed right to change the name.